tools/run_tests/run_tests.py

```diff
@@ -1590,8 +1590,15 @@ def make_jobspec(cfg, targets, makefile='Makefile'):
         return [
             jobset.JobSpec(
                 [
-                    'cmake', '--build', '.', '--target',
-                    '%s' % target, '--config', _MSBUILD_CONFIG[cfg]
+                    'cmake',
+                    '--build',
+                    '.',
+                    '--target',
+                    '%s' % target,
+                    '--config',
+                    _MSBUILD_CONFIG[cfg],
+                    '--',
+                    '/maxcpucount:%d' % args.jobs,
```

Comment:
Sorry for delay, but yeah as far as I know this is specific to msbuild.

I've sped things up locally/personally with this change, but it might not be the most portable way to do this as noted. And actually, it looks like cmake 3.12 [has added such a portable parrallel build feature](https://cmake.org/cmake/help/v3.12/release/3.12.html) - I haven't tried this but that might be a better way to go.

Reply:
heads up that there's still something funny with this PR on portability tests. Also, the ninja option sounds very interesting